README.md

```diff
@@ -1519,6 +1527,22 @@ bazel startup flag `--loading_phase_threads=1` in your bazel invocation.
            include CPU features such as <code>["sse4", "aes"]</code>.</p>
       </td>
     </tr>
+    <tr>
+      <td><code>puller_darwin</code></td>
+      <td>
+        <p><code>label; optional</code></p>
+        <p>A Mac 64-bit binary that implements the functionality provided by
```

Comment:
nit - add: "Visible for testing purposes" here and below

Reply:
Done